### PR TITLE
Kubernetes Docs overview page updates

### PR DESCRIPTION
**Pull request by tonykelly-octopus**
Updating Kubernetes Overview page on Docs.

**Before**
image

**After**
image

### CODE REVIEW

**Review by steve-fenton-octopus — APPROVED** [review]
LGTM

**Comment by tonykelly-octopus** [comment]
We're still reviewing this one with a few other folks so it may stay in this state for a while. Don't merge until everything is signed off (more a note for myself than anything!).

**Pull request merged**
